alembic/README.md

```diff
@@ -0,0 +1,57 @@
+# Generic single-database configuration.
+
+PyWPS uses [alambic](https://alembic.sqlalchemy.org/en/latest/tutorial.html) for
+database configuration.
+
+## To initialize new database and apply all changes
+
+Run
+
+```
+alembic upgrade head
+```
+
+You should get output similar to
+
+```
+INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
+INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
+INFO  [alembic.runtime.migration] Running upgrade eead25e2ad3a -> 01d5298bfc61, Add process column to Request
+```
+
+## To make changes in database structure
+
+1. Change ORM
+
+        Currently, all changes in the datase are  done in `pywps/dblog.py` - make the
```

Comment:
Currently, all changes in the database are done in `pywps/dblog.py` - make the
(typo in database and extra space)

Reply:
done.